### PR TITLE
chore(bridge-ui-v2): Update tooltips for Processing fee and Recipient address

### PR DESCRIPTION
**Pull request by xiaodino**
Before
Screen Shot 2023-08-16 at 3 13 34 PM

After
Screen Shot 2023-08-16 at 9 45 18 PM

### CODE REVIEW

**Comment by cyberhorsey** [comment]
Should these still be white even on dark mode?

File: packages/bridge-ui-v2/src/components/Activities/MobileDetailsDialog.svelte [hunk 1]
```diff
@@ -60,7 +60,7 @@
           <h4 class="text-secondary-content">{$t('activities.header.explorer')}</h4>
           <a
             href={`${chainUrlMap[Number(selectedItem.srcChainId)].explorerUrl}/tx/${selectedItem.hash}`}
-            target="_blank">
+            target="noopener">
```
Comment (KorbinianK) [inline packages/bridge-ui-v2/src/components/Activities/MobileDetailsDialog.svelte:63]: noopener is not a valid target. noopener is a rel attribute keyword.

[see Mozilla docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/noopener)

Comment (xiaodino) [reply packages/bridge-ui-v2/src/components/Activities/MobileDetailsDialog.svelte]: Reverted the change.

**Comment by xiaodino** [comment]
> Should these still be white even on dark mode?

It's not white. The background color is ```bg-primary-base-background```, which will be updated to another color in the light mode.

**Review by KorbinianK — DISMISSED** [review]

**Review by KorbinianK — APPROVED** [review]

**Review by dantaik — APPROVED** [review]

**Pull request merged**